src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/translation.html.twig

```diff
@@ -92,6 +93,14 @@
                     <code>{{ message.id }}</code>
                     {% if message.count > 1 %}<br><small style="color: gray;">(used {{ message.count }} times)</small>{% endif %}
                 </td>
+                <td>
+                    {% for key, value in message.parameters %}
+                        <code>{{ key }} = {{ value }}</code><br />
```

Comment:
this will add an extra ``<br>`` after the last parameter when there is no pluralization